### PR TITLE
ISPN-8255 Smoke test suite for hibernate-cache module

### PR DESCRIPTION
**Pull request by mgencur**
https://issues.jboss.org/browse/ISPN-8255

### CODE REVIEW

File: commons-test/src/main/java/org/infinispan/commons/test/categories/Smoke.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package org.infinispan.commons.test.categories;
+
+/**
+ * {@link org.junit.experimental.categories.Category} tag for smoke tests.
+ */
+public class Smoke {
```
Comment (rvansa) [inline commons-test/src/main/java/org/infinispan/commons/test/categories/Smoke.java]: Is there any advantage to using class/final class/interface as the category class? JUnit wiki uses interface in the example code.

Comment (mgencur) [reply commons-test/src/main/java/org/infinispan/commons/test/categories/Smoke.java]: Both a class and interface work. This is just a marker that the test class annotated with this type should be picked by the test suite runner. I don't see any difference.

Comment (mgencur) [reply commons-test/src/main/java/org/infinispan/commons/test/categories/Smoke.java]: Thinking about it a little more. We can build a hierarchy of categories with both interfaces and non-final classes.

**Comment by galderz** [comment]
Let's wait for CI, the rest looks good to me.

**Comment by rvansa** [comment]
CI looks good, integrating.

**Pull request merged**